### DISCUSSION TITLE
STM32f7 Discovery Board and FreeRTOS

### DISCUSSION THREAD

**Discussion by saul01** [micro-ROS/micro_ros_setup#283] [Ideas]
Hi,
I'd like to contribute to the project by adding support for the SMT32F7 (Cortex M7) and FreeRTOS, where do I get started?
Thank you.

Saul

**Comment by pablogs9** [comment]
Do you have the port done? You can contribute it with a PR

**Comment by pablogs9** [comment]
Do you have the port done? You can contribute it with a PR
